packages/runtime/test/src/logicsig.ts

```diff
@@ -1,27 +1,66 @@
 import { decodeAddress, multisigAddress, MultisigMetadata } from "algosdk";
 import { assert } from "chai";
-
+const { convert } = require("@algo-builder/algob");
 import { AccountStore } from "../../src/account";
 import { RUNTIME_ERRORS } from "../../src/errors/errors-list";
 import { Runtime } from "../../src/runtime";
 import { useFixture } from "../helpers/integration";
 import { expectRuntimeError } from "../helpers/runtime-errors";
+const { types } = require("@algo-builder/web");
 
 const programName = "escrow.teal";
 const multiSigProg = "sample-asc.teal";
+const crowdFundEscrow = "crowdFundEscrow.teal";
 
 describe("Logic Signature", () => {
 	useFixture("escrow-account");
 	let john: AccountStore;
 	let bob: AccountStore;
 	let runtime: Runtime;
 	let johnPk: Uint8Array;
+	let goal: number;
+	let crowdfundApprovalFileName: string;
+	let crowdFundClearFileName: string;
+	let now: Date;
+	let beginDate: Date;
+	let endDate: Date;
+	let fundCloseDate: Date;
+	let creationArgs: Array<{}>;
+	let appDefinition: any;
+	let applicationId: any
 
 	before(() => {
+		goal = 0.01e6;
+		crowdfundApprovalFileName = "crowdFundApproval.teal";
+		crowdFundClearFileName = "crowdFundClear.teal";
 		john = new AccountStore(10);
-		bob = new AccountStore(100);
+		bob = new AccountStore(10e6);
```

Comment:
Why do you set bob's account balance from 100 to 10e6?

Reply:
more balance is needed